go.mod

```diff
@@ -57,7 +57,23 @@ require (
 )
 
 replace (
+	github.com/decred/dcrdata/api/types => ./api/types
 	github.com/decred/dcrdata/blockdata => ./blockdata
+	github.com/decred/dcrdata/db/cache => ./db/cache
+	github.com/decred/dcrdata/db/dbtypes => ./db/dbtypes
```

Comment:
All this replace statements seem to be necessary so that tests for all this modules can be run.